install_config/install/advanced_install.adoc

```diff
@@ -1181,6 +1181,139 @@ xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#i
+catalog.
+
+To enable the service catalog, add the following in your inventory file's
+`[OSEv3:vars]` section:
+
+----
+openshift_enable_service_catalog=true
+ifdef::openshift-enterprise[]
+openshift_service_catalog_image_prefix=openshift3/ose-
+endif::[]
+ifdef::openshift-origin[]
+openshift_service_catalog_image_prefix=openshift/origin-
+openshift_service_catalog_image_version=latest
+endif::[]
+----
+
+When the service catalog is enabled, the web console shows the updated landing
+page but still uses the normal image stream and template behavior. To deploy and
+register the provided service brokers, see
+xref:configuring-ansible-service-broker[Configuring the Ansible Service Broker]
+and xref:configuring-template-service-broker[Configuring the Template Service Broker].
+
+[[configuring-ansible-service-broker]]
+=== Configuring the Ansible Service Broker
+
+[NOTE]
+====
+The Ansible service broker is currently a Technology Preview feature.
+====
+
+If you have xref:enabling-service-catalog[enabled the service catalog], you can
+also enable the
+xref:../../architecture/service_catalog/ansible_service_broker.adoc#arch-ansible-service-broker[Ansible service broker] (ASB).
+
+The ASB deploys its own etcd instance separate from the etcd used by the rest of
+the {product-title} cluster. The ASB's etcd instance requires separate storage
+using persistent volumes (PVs) to function, as well as for the Ansible playbook
+bundles (APBs) that are created using the ASB. If no PV is available, etcd will
+wait until the PV can be satisfied. The ASB application will enter a `CrashLoop`
+state until its etcd instance is available.
+
+[NOTE]
+====
+The following example shows usage of an NFS host to provide the required PVs,
+but
+xref:../../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[other persistent storage providers] can be used instead.
+====
+
+To enable the ASB and define its storage requirements:
+
+. Before running the installer, set up an NFS host.
+
+. Add the following in your inventory file's `[OSEv3:vars]` section:
++
+----
+ifdef::openshift-enterprise[]
+ansible_service_broker_image_prefix=openshift3/
+ansible_service_broker_registry_url="registry.access.redhat.com"
+endif::endif::[]
+ifdef::openshift-origin[]
+ansible_service_broker_image_prefix=openshift/
+ansible_service_broker_registry_url="registry.access.redhat.com"
+endif::[]
+
+openshift_hosted_etcd_storage_kind=nfs
+openshift_hosted_etcd_storage_nfs_options="*(rw,root_squash,sync,no_wdelay)"
+openshift_hosted_etcd_storage_host=<nfs_host> <1>
+openshift_hosted_etcd_storage_nfs_directory=/path/to/export
+openshift_hosted_etcd_storage_volume_name=etcd <2>
+openshift_hosted_etcd_storage_access_modes=['ReadWriteOnce']
+openshift_hosted_etcd_storage_volume_size: 1G
+openshift_hosted_etcd_storage_labels={'storage': 'etcd'}
+----
+<1> Set the IP or host name of your NFS host.
+<2> If link:https://bugzilla.redhat.com/show_bug.cgi?id=1467574[*BZ#1467574*] is
```

Comment:
I think this is actually already closed (https://bugzilla.redhat.com/show_bug.cgi?id=1467574), so we might not need to include this.